File: sample/src/main/java/ru/surfstudio/standard/ui/base/dagger/ActivityScreenModule.java
package ru.surfstudio.standard.ui.base.dagger;

import dagger.Module;
import ru.surfstudio.android.core.ui.base.dagger.CoreActivityScreenModule;
import ru.surfstudio.standard.ui.base.error.ErrorHandlerModule;

@Module(includes = {CoreActivityScreenModule.class, ErrorHandlerModule.class})
public class ActivityScreenModule {
}

File: template/src/main/java/ru/surfstudio/standard/ui/base/dagger/ActivityScreenModule.kt
package ru.surfstudio.standard.ui.base.dagger

import dagger.Module
import ru.surfstudio.android.core.mvp.dagger.CoreActivityScreenModule
import ru.surfstudio.android.mvp.dialog.dagger.DialogNavigatorForActivityModule
import ru.surfstudio.standard.ui.base.error.ErrorHandlerModule

@Module(includes = arrayOf(
        CoreActivityScreenModule::class,
        ErrorHandlerModule::class,
        DialogNavigatorForActivityModule::class))
class ActivityScreenModule